tools/releaseTools.psm1

```diff
@@ -341,21 +341,31 @@ function Get-ChangeLog
     PrintChangeLog -clSection $clBreakingChange -sectionTitle 'Breaking Changes'
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Feature tests for Get-Counter cmdlet.Get-Counter CounterSet tests.Can specify the sample interval
```powershell
Expected the actual value to be greater than or equal to 10, but got 9.9944701.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Diagnostics\Get-Counter.Tests.ps1: line 176
176:             ($endTime - $startTime).TotalSeconds | Should -Not -BeLessThan ($counterCount * $sampleInterval)
```